R/ltcad.R

```diff
@@ -0,0 +1,133 @@
+#' @method epidist_prepare epidist_ltcad
+#' @family ltcad
+#' @export
+epidist_prepare.epidist_ltcad <- function(data) {
+  id <- obs_t <- obs_at <- NULL
```

Comment:
I generally prefer to do the NULLing via global variables in utils but I don't feel strongly.

Reply:
We have an existing issue for that. This was to fix the warnings as they appear here. I'll look into the two options on that issue (I don't know how the global variables option works at the moment).